sdk/translation/azure-ai-translation-text/azure/ai/translation/text/_patch.py

```diff
@@ -56,6 +56,29 @@ def on_request(self, request: PipelineRequest) -> None:
         request.http_request.headers["Ocp-Apim-Subscription-Key"] = self.credential.key
         request.http_request.headers["Ocp-Apim-Subscription-Region"] = self.credential.region
 
+class TranslatorAADCredential:
+    """Credential for Translator Service when using AAD authentication."""
```

Comment:
Can you also add docs here for the properties?

Reply:
Sure